final_task/rss_reader/rss_parser.py

```diff
@@ -0,0 +1,274 @@
+import sys
+import feedparser
+import json
+import logging
+import requests
+from yattag import Doc
+from fpdf import FPDF
+import os
+from database_functions import put_into_db, check_existance, json_from_cashe
+from database_functions import open_database, printing
+
+class RssParser():
+    """ Parses news from website"""
+    def __init__(self, *args):
+        """ Inits arguments"""
+
+        self.url = args[0]
+        self.limit = args[1]
+        self.jsonFormat = args[2]
+        self.date = args[3]
+        self.to_html = args[5]
+        self.path = args[4]
+        self.to_pdf = args[6]
```

Comment:
Do you actually need class for this? You could pass args as an argument and do not use class at all.